Tangra 3/Video/FITS/frmSortFitsFiles.cs

```diff
@@ -200,12 +210,13 @@ private void timer1_Tick(object sender, EventArgs e)
             m_SortedByTimeStamp = true;
             if (m_FilesWithExposure == 0 && m_FilesWithoutExposure > 0)
             {
-                MessageBox.Show("None of the FITS files have a saved exposure and they have been ordered by file name!", "Tangra", MessageBoxButtons.OK, MessageBoxIcon.Warning);
+                // No longer need this warning - it is handled by FITS Time Headers Dialog
```

Comment:
May be delete the commented out MessageBox and update the inserted comment to not mention the warning but rather describe in which case we end up here :)